sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
+## 1.0.1 (Unreleased)
 
 ## 1.0.0-beta.6 (2021-02-26)
 ### Breaking Changes
-- Change ChatParticipant id type from CommunicationUserIdentifier to String
```

Comment:
Shouldn't modify this. This was last release. You may rephrase but shouldn't add content. Add the the latest version "Breaking Changes" instead.

Reply:
Right, it is removed by merge because both used beta.6 version. Let me update it back.